src/enclave/rpc_sessions.h

```diff
@@ -97,7 +97,7 @@ namespace enclave
       auto search = sessions.find(id);
       if (search == sessions.end())
       {
-        LOG_FAIL_FMT("Replying to unknown session {}", id);
+        LOG_FAIL_FMT("Refusing to reply to unknown session {}", id);
```

Comment:
Unrelated to this PR: this should probably be a `LOG_DEBUG_FMT()` as this can happen often in a normal scenario (e.g. if a client disconnects before it has received the response to its forwarded request).

Reply:
Yeah, you're right, I'll make it a `_DEBUG_`.